React/Views/RCTSlider.m

```diff
@@ -32,4 +32,17 @@ - (void)setMaximumValue:(float)maximumValue
   super.value = _unclippedValue;
 }
 
+-(void)setTrackImage: (NSString *) trackImageSrc
+{
+
+  UIImage *sliderTrackImage = [UIImage imageNamed: trackImageSrc];
```

Comment:
Instead of `[UIImage imageNamed:]` use `[RCTConvert UIImage:]` instead, which is much more flexible.